### ISSUE TITLE
Search placeholder not disappearing

### ISSUE THREAD

**Issue by ReverseWineSnob** [algolia/algoliasearch-wordpress#811]
### What did you expect to happen?
The search placeholder text should be removed when a user clicks on the search box.

### What happened instead?
The search placeholder text stays, meaning the user needs to delete it out

### How can we reproduce this behavior?
Visit https://www.reversewinesnob.com/ and click on the search box on the top of the page

Can you provide a link to a page which shows this issue?
https://www.reversewinesnob.com/

### Technical info
* WordPress version: 5.0.1
* Algolia Search plugin version: 2.11.2

This was working correctly until today. Not sure if it is related to the Wordpress 5.0.1 update or not.

**Comment by rayrutjes** [comment]
Hello,

It seems like the `value` is set to the `placeholder` value.
Could it be that the default `query` is set to `Search for a wine, region, variety & more...` somehow?
I think by default we set the field value if we manage to get it from WordPress.
You can see that behavior here: https://www.reversewinesnob.com/?s=t

This leads me to think the default value we fetch from WordPress is your placeholder.

For now you are the first person reporting this. Let me know if you discover anything here.

**Comment by ReverseWineSnob** [comment]
Not sure how the placeholder would become the value all of a sudden unless something wonky is going on with 5.0.1, which could very well be case. Up until yesterday the "Search for a wine, region, variety & more..." (which is the placeholder in my functions.php) would be removed automatically when you clicked on the search box.

In any case, in playing around with this there are other weird things happening. For example, when trying to edit the functions.php (I wanted to see what would happen if I removed the placeholder) I get the following: "Unable to communicate back with site to check for fatal errors, so the PHP change was reverted. You will need to upload your PHP file change by some other means, such as by using SFTP." In Googling this it appears it is a plugin conflict:
In most cases, the issue lies with an open “session_start()” function being called unconditionally, meaning that the plugin or theme is trying to initiate PHP Sessions everywhere on the site, instead of strictly where it is needed.
Not sure if the two issues are related or not but hopefully they'll both be resolved with the next WP update.
